components/Modal/script.js

```diff
@@ -0,0 +1,9 @@
+const modalBtn = document.querySelector("#display-modal");
+const xBtn = document.querySelector("#x-btn");
```

Comment:
Use better and meaningful name for variable like you can name this variable to closeButton or closeBtn